src/content/organizations.json

```diff
@@ -70,10 +70,21 @@
         "infoUrl": "https://www.givinggreen.earth/us-policy-change-research/evergreen-collaborative:-recommendation",
         "shortDescription": "Designar policyförslag, sätter agendan för större miljöorganisationer och bedriver påverkansarbete med extra slagkraft under 2021-2022."
       },
-    
       {
-        "id": "25",
-        "name": "Good Food Institute",
+        "id": "27",
```

Comment:
Can we separate these new organizations out into a new PR? Or why are they included in this PR?

Reply:
They are not part of the PR, I based this PR on the main branch which have the changes :) I can definitely remove them.